### DISCUSSION TITLE
openssl 1.1.1v cpython-bin-deps for windows

### DISCUSSION THREAD

**Discussion by Ganeshkumhar1** [openssl/openssl#21806] [Q&A]
Hi ,

could someone please help when this will be available on below url ?
https://github.com/python/cpython-bin-deps/tree/openssl-bin-1.1.1v

it's been 1 month openssl-bin-1.1.1v released.

**Comment by t8m** [comment]
As this is not a repository the OpenSSL team maintains I do not think it makes sense to post the question here.
